src/chat/dto/post-chat.dto.ts

```diff
@@ -1,20 +1,50 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import {
+  IsMongoId,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+} from 'class-validator';
+import mongoose from 'mongoose';
 
 export class PostChatDto {
+  @IsMongoId()
+  @IsNotEmpty()
+  @ApiProperty({
+    example: '650bde3798dd4c34439c30dc',
+    description: '채팅을 전송하는 채팅방 id',
+  })
+  roomId: mongoose.Types.ObjectId;
+
   @ApiProperty({
     example: '안녕하세요',
     description: '채팅 내용',
   })
   @IsString()
-  @IsNotEmpty()
+  @IsOptional()
   content: string;
 
   @ApiProperty({
     example: '1',
+    description: '채팅을 보내는 유저 아이디',
+  })
+  @IsNumber()
+  @IsNotEmpty()
+  senderId: number;
```

Comment:
```suggestion
  @IsNumber()
  senderId: number;
```
기본적으로 @IsNumber, @IsString 같은 데코레이터는 IsNotEmpty에서 수행하는 validation 을 수행하게됩니다.
둘다 써야하는 경우는 프론트엔드측에 좀 더 자세한 에러메시지를 주는 경우에만 사용하는게 좋을거같아요.